models/DataObject/ClassDefinition/Data/ExternalImage.php

```diff
@@ -20,7 +20,7 @@
 use Pimcore\Model\DataObject;
 use Pimcore\Model\DataObject\ClassDefinition\Data;
 
-class ExternalImage extends Data implements ResourcePersistenceAwareInterface, QueryResourcePersistenceAwareInterface
```

Comment:
why did you drop the QueryResourcePersistenceAwareInterface interface ?

Reply:
copy paste miss..reverted it